package.json

```diff
@@ -90,7 +90,7 @@
 			"@oclif/plugin-help"
 		],
 		"hooks": {
-			"init": "./src/lib/hooks/init"
+			"init": "./lib/lib/hooks/init"
```

Comment:
With this change, oclif always loads it from the src/lib directory at dev time, even if the hook exists in lib/lib. Not what I expected exactly, but it works, and won't require `prepack` during dev.